.wuf/generator/templates/mobile/Component/Component.tsx

```diff
@@ -35,13 +35,13 @@ export class {%ComponentName%} extends React.Component<{%ComponentName%}Props, S
     return (
       <TPAComponentsConsumer>
         {({ mobile }) => (
-          <div {...styles('root', { mobile }, rest)} data-mobile={mobile}>
-            <Text {...styles('number', { even: isEven, odd: !isEven })}>
+          <div className={st(classes.root, { mobile }, rest)} data-mobile={mobile}>
+            <Text className={st(classes.number, { even: isEven, odd: !isEven })}>
```

Comment:
I remember it can be used on non-root elements if it has a state that needs to be sent.
@barak007 What is the convention? on a non-root stateful element, should I use the style function with the states or should I use the cssStates function?

Reply:
In any case, the template shouldn't have this form of stylable state.
It's not something we need for a new component, and it's not something we want to encourage.
But you can ignore this comment.
We need to do lots of cleaning in the generator template.
We'll do it in another PR